frontend/src/component/admin/auth/OidcAuth/OidcAuth.tsx

```diff
@@ -45,10 +46,10 @@ type State = typeof initialState & {
 export const OidcAuth = () => {
     const { setToastData, setToastApiError } = useToast();
     const { uiConfig } = useUiConfig();
+    const { oidcLocked } = uiConfig;
     const [data, setData] = useState<State>(initialState);
     const { config } = useAuthSettings('oidc');
     const { updateSettings, errors, loading } = useAuthSettingsApi('oidc');
-
     useEffect(() => {
```

Comment:
Poor empty line got yeeted, but I think it helps with readability / separation of logic blocks 😄 
```suggestion
    
    useEffect(() => {
```